dev/MenuFlyout/MenuFlyout_19h1_themeresources.xaml

```diff
@@ -164,7 +165,7 @@
                                 </VisualState>
                                 <VisualState x:Name="PointerOver">
                                     <VisualState.Setters>
-                                        <Setter Target="LayoutRoot.(RevealBrush.State)" Value="PointerOver" />
+                                        <Setter Target="LayoutRoot.(media:RevealBrush.State)" Value="PointerOver" />
```

Comment:
Why we didn't make the change to MenuFlyout_rs2_? Is it possible the merge it to only one file? We already did this to a lot of controls like CalendarView, checkbox

Reply:
Reveal was added to MenuFlyout in RS4 so MenuFlyout_rs2 doesn't have those VisualState setters.

I think you are right, we've already done the merge for many controls so I'll merge MenuFlyout too to make things simpler.

Reply:
Merged them to one file with conditional xaml